### PR TITLE
Module DOCUMENTATION should match argspec

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY
Large update of many modules so that DOCUMENTATION option name and
aliases match those defined in the argspec.
    
Issues identified by https://github.com/ansible/ansible/pull/34809
    
In addition to many typos and missing aliases, the following notable
changes were made:
    
* Create`module_docs_fragments/url.py` for `url_argument_spec`
*` dellos*_command` shouldn't have ever had `waitfor` (incorrectly copied deprecated aliases)
* `ce_aaa_server_host.py` `s/raduis_server_type/radius_server_type/g`
* `junos_lldp` enable should be part of `state`.

##### ISSUE TYPE
 - Docs Pull Request

### CODE REVIEW

**Comment by gundalow** [comment]
bot_status

File: lib/ansible/modules/cloud/amazon/lambda_alias.py [hunk 1]
```diff
@@ -52,9 +52,9 @@
 requirements:
     - boto3
 extends_documentation_fragment:
-    - aws
-
-'''
+  - aws
+  - ec2
+ '''
```
Comment (sivel) [inline lib/ansible/modules/cloud/amazon/lambda_alias.py]: Is this extra space accidental?

File: lib/ansible/modules/cloud/amazon/lambda_facts.py [hunk 2]
```diff
@@ -52,9 +52,9 @@
 requirements:
     - boto3
 extends_documentation_fragment:
-    - aws
-
-'''
+  - aws
+  - ec2
+ '''
```
Comment (sivel) [inline lib/ansible/modules/cloud/amazon/lambda_facts.py]: Extra space

File: lib/ansible/modules/cloud/amazon/lambda_policy.py [hunk 3]
```diff
@@ -91,9 +91,9 @@
 requirements:
     - boto3
 extends_documentation_fragment:
-    - aws
-
-'''
+  - aws
+  - ec2
+ '''
```
Comment (sivel) [inline lib/ansible/modules/cloud/amazon/lambda_policy.py]: Extra space

File: lib/ansible/modules/cloud/digital_ocean/digital_ocean_domain.py [hunk 4]
```diff
@@ -35,6 +35,7 @@
   id:
     description:
      - Numeric, the droplet id you want to operate on.
+     aliases: ['droplet_id']
```
Comment (sivel) [inline lib/ansible/modules/cloud/digital_ocean/digital_ocean_domain.py]: over indented

File: lib/ansible/modules/packaging/os/urpmi.py [hunk 5]
```diff
@@ -41,6 +41,7 @@
       - Corresponds to the C(--no-recommends) option for I(urpmi).
     type: bool
     default: 'yes'
+    aliases@ ['no-recommends']
```
Comment (sivel) [inline lib/ansible/modules/packaging/os/urpmi.py]: `s/@/:/`

**Comment by sivel** [comment]
In addition to my comments, from the last build (ignoring version_added comments which are invalid):

```
lib/ansible/modules/cloud/digital_ocean/digital_ocean_certificate.py:48:1: E302 DOCUMENTATION is not valid YAML
lib/ansible/modules/cloud/digital_ocean/digital_ocean_domain.py:38:6: E302 DOCUMENTATION is not valid YAML
lib/ansible/modules/packaging/os/urpmi.py:45:3: E302 DOCUMENTATION is not valid YAML
```

**Review by mattclay — APPROVED** [review]
Looks good, aside from what @sivel already noted.

**Comment by gundalow** [comment]
Fixed all the issues
CI issues are `version_added` which will go away once this is merged

**Pull request merged**